test/amd.html

```diff
@@ -18,6 +18,7 @@
 	});
 
 	require( [ "qunit", "./amd" ], function( QUnit, tests ) {
+		QUnit.warning.off();
```

Comment:
these specific tests should not worry about triggering it off. That should occur only for the main test or anything else that are still covering deprecated API.